src/interface/meta.thrift

```diff
@@ -455,6 +459,20 @@ struct ListEdgeIndexesResp {
     3: list<common.IndexItem>    items,
 }
 
+struct RebuildIndexReq {
+    1: common.GraphSpaceID space_id,
+    2: string              index_name,
+    3: common.SchemaID     schema_id,
+    4: common.SchemaVer    version,
+}
+
+struct RebuildIndexResp {
```

Comment:
I have a question, why not use ExecResp?

Reply:
make sense